cloud_dashboard/src/service/utility.ts

```diff
@@ -66,6 +66,58 @@ const roundNumber = (value: number, digit: number) => {
   return temp;
 };
 
+const convertKeyValueCrLfData = (data: string) => {
+  const temp = data.split('\n');
```

Comment:
@RyoYamashita765 This can be:

```suggestion
  const records = data.split('\n');
```